src/Endpoints/Keys.php

```diff
@@ -65,18 +65,32 @@ protected function fill(array $attributes): self
         $this->actions = $attributes['actions'];
         $this->indexes = $attributes['indexes'];
         if ($attributes['expiresAt']) {
-            $this->expiresAt = date_create_from_format('Y-m-d\TH:i:s\Z', $attributes['expiresAt']);
+            $this->expiresAt = $this->createDate($attributes['expiresAt']);
         }
         if ($attributes['createdAt']) {
-            $this->createdAt = date_create_from_format('Y-m-d\TH:i:s.vu\Z', $attributes['createdAt']);
+            $this->createdAt = $this->createDate($attributes['createdAt']);
         }
         if ($attributes['updatedAt']) {
-            $this->updatedAt = date_create_from_format('Y-m-d\TH:i:s.vu\Z', $attributes['updatedAt']);
+            $this->updatedAt = $this->createDate($attributes['updatedAt']);
         }
 
         return $this;
     }
 
+    protected function createDate($attribute): ?DateTime
```

Comment:
typehint $attribute here?

Reply:
My approach to this whole solution is not having any failure point, and assigning a sane value if unexpected data is provided.

I don't know how safe this change would be considering that approach, but that is for @brunoocasali & co to decide.